src/main/scala/io/iohk/ethereum/metrics/AppJmxConfig.scala

```diff
@@ -0,0 +1,11 @@
+package io.iohk.ethereum.metrics
+
+import io.micrometer.jmx.JmxConfig
+
+class AppJmxConfig extends JmxConfig {
+  def get(key: String): String = null
```

Comment:
Do we need that?

Reply:
yes, (override is missing, i will add it)